.agents/skills/superposition/reference/documentation/other/client_context_aware_configuration.md

```
@@ -0,0 +1,380 @@
+## Rust
+
+The rust client have a client factory that helps you work with multiple clients connected to different tenants
+
+### Client Factory Methods Reference
+
+#### Create Client
+
+Create a client in the factory. You can chose to use the result to check for errors faced by the Client Factory while creating your client, it is not mandatory to consume the `Ok` value.
+
+##### Function definition
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)